### PR TITLE
HPC: use mount_nfs sub

### PR DESCRIPTION
**Pull request by schlad**
Replace direct mount of the NFS and use sub provided by the hpcbase

Verification run: http://10.160.65.14/tests/11465
http://10.160.65.14/tests/11465#step/slurm_master_adv/26

### CODE REVIEW

File: tests/hpc/slurm_master_backup_db.pm [hunk 1]
```diff
@@ -23,21 +23,14 @@ sub run {
...
-    systemctl 'start rpcbind';
-    record_info('show mounts aviable on the supportserver', script_output('showmount -e 10.0.2.1'));
-    assert_script_run("mkdir -p /shared/slurm");
-    assert_script_run("chown -Rcv slurm:slurm /shared/slurm");
-    assert_script_run("mount -t nfs -o nfsvers=3 10.0.2.1:/nfs/shared /shared/slurm");
+    $self->mount_nfs();
 
-    # provision HPC cluster, so the proper rpms are installed
-    # and proper services are enabled and started
-    zypper_call('in slurm slurm-munge');
```
[the remark this answers is not in the document]
Comment (schlad) [reply tests/hpc/slurm_master_backup_db.pm]: yup, just for better code organization. In fact following your earlier remark in a different PR, that in make sense to group zypper in.

File: tests/hpc/slurm_master_backup_db.pm [hunk 2]
```diff
@@ -23,21 +23,14 @@ sub run {
     my $self  = shift;
     my $nodes = get_required_var("CLUSTER_NODES");
     $self->prepare_user_and_group();
+    zypper_call('in nfs-client rpcbind');
```
Comment (jlausuch) [inline tests/hpc/slurm_master_backup_db.pm]: would it make sense also to move `zypper in nfs-client` to mount_nfs?

Comment (schlad) [reply tests/hpc/slurm_master_backup_db.pm]: sure, why not actually.

**Comment by schlad** [comment]
@jlausuch v2 pushed.

File: tests/hpc/slurm_master_backup_db.pm (hunk 2 above)
Comment (schlad) [reply tests/hpc/slurm_master_backup_db.pm]: changed.

**Pull request merged**